herbert_core/classes/MPIFramework/@ClusterParpoolWrapper/ClusterParpoolWrapper.m

```diff
@@ -114,6 +114,18 @@
             
             cl  = parcluster();
             
+            % By default Matlab only utilises physical cores; enable use of
+            % logical cores if required
+            if cl.NumWorkers < obj.n_workers
+                [n_physical_cores, n_logical_cores] = get_num_cores();
+                if obj.n_workers <= n_logical_cores
```

Comment:
does there need to be a `n_workers > n_logical_cores` else block that displays an (friendly) error, rather than waiting for the inevitable exception

Reply:
The block below does this